testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/security/xacml/EjbXACMLAuthorizationModuleTestCase.java

```diff
@@ -160,7 +163,13 @@ private static JavaArchive createJar(final String archiveName, final String secu
                 .addAsResource(EjbXACMLAuthorizationModuleTestCase.class.getPackage(),
                         XACMLTestUtils.TESTOBJECTS_POLICIES + "/ejb-xacml-policy.xml", "xacml-policy.xml")
                 .addAsManifestResource(EjbXACMLAuthorizationModuleTestCase.class.getPackage(),
-                        XACMLTestUtils.TESTOBJECTS_CONFIG + "/jboss-ejb3.xml", "jboss-ejb3.xml");
+                        XACMLTestUtils.TESTOBJECTS_CONFIG + "/jboss-ejb3.xml", "jboss-ejb3.xml")
+                .addAsResource(createPermissionsXmlAsset(
+                        new RuntimePermission("accessDeclaredMembers"),
+                        new RuntimePermission("getClassLoader"),
+                        new ElytronPermission("getSecurityDomain"),
+                        new ReflectPermission("suppressAccessChecks")
```

Comment:
mentioned RuntimePermissions should not be required to be assigned to client code - I am going to add required privileged block into XACML as discussed in https://issues.jboss.org/browse/WFLY-10190